exonum/src/explorer/mod.rs

```diff
@@ -76,6 +76,24 @@ pub enum TxStatus {
     },
 }
 
+/// Range information. Range borders are included.
+#[derive(Debug, Serialize, Deserialize)]
+pub struct Range {
```

Comment:
Can't we use `std::Range`?

Reply:
https://doc.rust-lang.org/std/ops/struct.RangeInclusive.html this one?

Reply:
https://doc.rust-lang.org/std/ops/struct.Range.html this? Or you need inclusive one?

Reply:
By the logic of the block exploration, it makes sense to work with closed intervals.

Reply:
Sadly, I don't think it's possible as it's nightly-only API atm.